scripts/retrieve_training_metrics.py

```diff
@@ -26,47 +114,16 @@ def main():
         "version",
         nargs="?",
         help="Which bugbug version should we retrieve training metrics from.",
```

Comment:
I think for the version we should default to "latest". The main use case is getting how metrics evolve over time, even while the versions of bugbug change